src/__tests__/postgres/integration.test.ts

```diff
@@ -394,4 +394,62 @@ describeMaybeSkip('integration (postgres)', () => {
     expect(errors).toBeUndefined()
     expect(data?.films.some((f: any) => f.sequel && f.sequel.id)).toBe(true)
   })
+
+  test('subscriptions', async () => {
```

Comment:
suggestion: This test can be removed.

This test is extraneous because it's not testing anything specific to the library.